core/client/ui/onboarding.hbs.html

```diff
@@ -0,0 +1,15 @@
+<template name="onboarding">
+    <div class="main">
+        <img class="logo" src='lemverse.png' alt="logo" />
+        <div class="container">
+            <div class="container-title">
+                Activate your camera and microphone to continue
+            </div>
+            <div class="explanation-container">
+                <img class="permission-img" src='screenshot-user-media.png' alt="User permission popup" />
```

Comment:
Until we support other languages than English, `screenshot-user-media.png` shouldn't show a French UI. :)